tests/Makefile

```diff
@@ -7,6 +7,12 @@ EXCLUDED += \
   $(NULL)
 endif
 
+ifeq ($(JSR_179),0)
+EXCLUDED += \
+  "./javax/microedition/location/*" \
+  $(NULL)
+endif
```

Comment:
Since you're only excluding a single additional directory, this could be simply:

```
ifeq ($(JSR_179),0)
EXCLUDED += "./javax/microedition/location/*"
endif
```